.circleci/config.yml

```
@@ -25,6 +33,19 @@ jobs:
                 command: ./node_modules/.bin/eslint src
                 when: always
 
+            - run:
+                name: "Build Node.js package"
+                command: npm run build:js
```

```suggestion
                command: npm run build:js-dev
```

This will catch the errors, but it is significantly faster (since it doesn't have to minify the code).